pkg/component/delete/delete.go

```diff
@@ -97,10 +73,10 @@ func (o DeleteComponentClient) getPod(componentName, appName string) (pod *corev
 		klog.V(1).Info("Component not found on the cluster.")
 		if kerrors.IsForbidden(err) {
 			klog.V(3).Infof("Resource for %s forbidden", componentName)
-			return pod, nil
+			return pod, err
```

Comment:
Thsi changes the behaviour described in the comment line 91/67. Are you sure you want to change this behaviour?

Reply:
Can't believe I missed that. I was focused on removing log.Status out of this pkg, but also wanted to ensure the correct error messages are printed, I forgot about the effect of this one. Thanks for catching this. :100: